README.md

```diff
@@ -146,139 +142,26 @@ JSON output:
 }
 ```
 
-## 🤖 Compatibility with MeiliSearch
-
-This package only guarantees the compatibility with the [version v0.15.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.15.0).
-
-## 🎬 Examples
-
-All HTTP routes of MeiliSearch are accessible via methods in this SDK.</br>
-You can check out [the API documentation](https://docs.meilisearch.com/references/).
-
-### Indexes
-
-#### Create an index <!-- omit in toc -->
-
-```go
-// Create an index with a specific uid (uid must be unique)
-resp, err := client.Indexes().Create(meilisearch.CreateIndexRequest{
-    UID: "books",
-})
-// Create an index with a primary key
-resp, err := client.Indexes().Create(meilisearch.CreateIndexRequest{
-    UID: "books",
-    PrimaryKey: "book_id",
-})
-```
-
-#### List all indexes <!-- omit in toc -->
-
-```go
-list, err := client.Indexes().List()
-```
-
-#### Get an index object <!-- omit in toc -->
-
-```go
-index, err := client.Indexes().Get("books")
-```
-
-### Documents
-
-#### Fetch documents <!-- omit in toc -->
-
-```go
-// Get one document
-var document map[int]interface{}
-err := client.Documents("books").Get("123", &doc)
-// Get documents by batch
-var list []map[int]interface{}
-err = client.Documents("books").List(ListDocumentsRequest{
-    Offset: 0,
-    Limit:  10,
-}, &list)
-```
-
-#### Add documents <!-- omit in toc -->
-
-```go
-documents := []map[string]interface{}{
-    {BookID: 90, Title: "Madame Bovary"},
-}
-
-upd_res, err := client.Documents("books").AddOrUpdate(documents)
-```
-
-Response:
-```json
-{
-    "updateId": 1
-}
-```
-With this `updateId` you can track your [operation update](#update-status).
-
-#### Delete documents <!-- omit in toc -->
-
-```go
-// Delete one document
-updateRes, err = client.Documents("books").Delete("123")
-// Delete several documents
-updateRes, err = client.Documents("books").Deletes([]string{"123", "456"})
-// Delete all documents /!\
-updateRes, err = client.Documents("books").DeleteAllDocuments()
-```
+#### Custom Search <!-- omit in toc -->
 
-### Update status
+All the supported options are described in [this documentation section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```

Comment:
```suggestion
All the supported options are described in [the search parameters documentation section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```

Reply:
```suggestion
All the supported options are described in [the search parameters section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```

Reply:
Your suggestion seems a little bit "heavy": WDYT of this one?
(I know I accepted yours in the Python SDK, but I realized just now I don't really like it 😅 I will change it of cours in the Python if you agree)

Reply:
Sure, it's lighter. Even if it doesn't specify what it is a 'section' of. But I still prefer it to the first one, so go! :)

Reply:
Or 

```suggestion
All the supported options are described in [the search parameters documentation](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```

Reply:
```suggestion
All the supported options are described in [the search parameters section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html) of the documentation.
```

Reply:
```suggestion
All the supported options are described in the [search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html) section of the documentation.
```